src/components/autocomplete/autocomplete.spec.js

```diff
@@ -1125,6 +1125,33 @@ describe('<md-autocomplete>', function() {
       element.remove();
     }));
 
+    it('should log a warning if the display text does not evaluate to a string',
+      inject(function($log) {
+        $log.reset();
```

Comment:
But that shouldn't trigger a new `$log.warn` call, since those were already executed.